.github/workflows/analyze.yml

```
@@ -0,0 +1,15 @@
+
+on:
+  workflow_call:
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Flutter
+        uses: ./.github/actions/setup
```

actionlint (1.7.4)

description is required in metadata of "Setup Flutter environment and get dependencies" action at "/home/jailuser/git/.github/actions/setup/action.yml"

(action)